src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -129,25 +129,16 @@ function getChatTabBrickRoad(policyID?: string): BrickRoad | undefined {
     // If policyID is undefined, then all reports are checked whether they contain any brick road
     const policyReports = policyID ? Object.values(allReports).filter((report) => report?.policyID === policyID) : Object.values(allReports);
 
-    let hasChatTabGBR = false;
-
-    const hasChatTabRBR = policyReports.some((report) => {
+    return policyReports.find((report) => {
         const brickRoad = report ? getBrickRoadForPolicy(report) : undefined;
-        if (!hasChatTabGBR && brickRoad === CONST.BRICK_ROAD_INDICATOR_STATUS.INFO) {
-            hasChatTabGBR = true;
-        }
-        return brickRoad === CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR;
+        // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+        return [CONST.BRICK_ROAD_INDICATOR_STATUS.INFO, CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR].includes(brickRoad as NonNullable<BrickRoad>);
     });
+}
 
-    if (hasChatTabRBR) {
-        return CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR;
-    }
-
-    if (hasChatTabGBR) {
-        return CONST.BRICK_ROAD_INDICATOR_STATUS.INFO;
-    }
-
-    return undefined;
+function getChatTabBrickRoad(policyID?: string): BrickRoad | undefined {
+    const report = getChatTabBrickRoadReport(policyID);
+    return report ? getBrickRoadForPolicy(report) : undefined;
```

Comment:
What happened to the previous logic? Are we maintaining it somewhere ?

Reply:
@fabioh8010 I got confused with my own code hehehe but I managed to restore the previous logic!

Thanks for this, great catch 😄